api/services/DeveloperService.js

```diff
@@ -9,14 +9,25 @@ class DeveloperService {
     return this.api
       .find(username)
       .then(function (user) {
-        if (user.type !== 'User') {
-          return Promise.reject({statusCode: 404});
-        }
 
         var profileInformation = _.pick(user, Object.keys(Profile.definition));
 
         delete profileInformation.id;
 
+        if (user.type === 'Organization') {
+          Organization.create({
```

Comment:
Move this logic into an organization service and import anew. Its an extra call, yes, but it's separated. I do like that you're not waiting for this to finish! That's good. The user doesn't have to wait for it.